lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUDriver.cs

```diff
@@ -243,6 +279,45 @@ public void OnNext(IFailedEvaluator value)
         }
 
         /// <summary>
+        /// This is to remove failed context from _activeContexts
+        /// More details will be implemented when working on REEF-1251
+        /// </summary>
+        /// <param name="value"></param>
+        private void RemovedFailedContext(IFailedEvaluator value)
```

Comment:
I'm ok with merging this. But I think we will find ourselves moving the same code around a lot in the comming PRs. Hence, I was wondering whether we should e.g. create and use a `ContextManager` here already, given that this PR adds large methods that deal with context management.